planter/planter.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: use like: ./planter.sh bazel build //cmd/...
+# NOTE: build kubernetes with: <path to test-infra>/planter.sh make bazel-build
+
+VERSION="0.1"
+IMAGE="gcr.io/k8s-testimages/planter:${VERSION}"
+# run our docker image as the host user with bazel cache and current repo
+DIR=$(git rev-parse --show-toplevel 2> /dev/null)
+# if not in a git repo, use $PWD
+DIR=${DIR:-"${PWD}"}
```

Comment:
This is a little unnecessary for our needs but it's only one line and it allows tinkering with the tool outside of a git repo. 

Also, if anyone starts using `planter` in the future we may want to add the ability to override some of these assumptions, but for now I think they're good (and necessary to function).

Reply:
I think you want to use `-w ${PWD}` so that I can do something like
```
$ cd prow/
$ ../planter/planter.sh test :all
```
and only test the things in `prow/`.

is this script ever going to be called outside of a git repo?

Reply:
I suppose not. We can always change it later, I will switch it this.